src/common/crypto/platform/openssl/crypto.cpp

```diff
@@ -825,11 +767,8 @@ error::Error PrivateKey::SaveToPEM(const string &private_key_path) {
 				+ "): " + GetOpenSSLErrorMessage());
 	}
 
-	// PEM_write_bio_PrivateKey_traditional will use the key-specific PKCS1
-	// format if one is available for that key type, otherwise it will encode
-	// to a PKCS8 key.
```

Comment:
I don't remember the details but this had a reason to be.

Are we removing it now because ED25519 does not have a PKCS1 (tradictional) formatting anyway?

Reply:
ED25519 doesn't support the traditional format, so we can't use `PEM_write_bio_PrivateKey_traditional()`

Reply:
Ok, It makes sense to change. I verified that in production code, the `Save` is only called after a `Generate`, so we know what are we saving.